Arc/App/Extensions/Project/Session+Extensions.swift

```diff
@@ -200,5 +200,13 @@ public extension Session {
 		})
 	}
 
-
+    func getCognitiveSurveyFor(surveyType:SurveyType) -> JSONData? {
```

Comment:
When the user does the tutorial for the first time, there are 2 data results that come back for a cognitive test.  In that case, the tutorial result is much smaller, so sorting by largest returns the correct test result.  

See https://sagebionetworks.jira.com/browse/DIAN-682